### PR TITLE
CD-161982 master:  add simple generic check

### PR DESCRIPTION
**Pull request by andy-edward**
Remove "StatsdCheck" to be replaced with a generic simple check.
Add new check type "SimpleGenericCheck" that is in the list of check types that will be executed by the /health action.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,17 +32,8 @@ The class, or in some cases, an instance can also be passed in.
 
 ### Available checks:
 Simple checks:
-* `SimpleHealthCheck::BasicStatusCheck` - returns "status": 1
+* `SimpleHealthCheck::BasicStatus` - returns "status": "ok"
```
Comment (edk) [inline README.md]: how will this affect existing clients of this gem?  I don't see any uses of it in my latest repo, but is it possible that another repo is directly referencing this constant?

**Comment by edk** [comment]
looks good so far, thank you!

my only potential concern is the impact to existing users of the gem. I'm not sure, so can you help me understand this aspect of the change?  Thanks :)

File: README.md (hunk 1 above)
Comment (andy-edward) [reply README.md]: @edk , I didn't change the way this behaves, just changed the doc to match what I was seeing in the result ```{"version":null,"status":"ok"}```, and the class name to match what I saw in the code.  Possible I missread, or missunderstood, I can remove that change in the doc if it is not correct.

Comment (edk) [reply README.md]: no, i think the response is correct, based on what we want to respond with.  it's just the change in ruby constant that i was concerned with.

Comment (edk) [reply README.md]: oh, this is a doc only change.  sorry, my bad, i totally space here.  looks good :D 👍

**Comment by edk** [comment]
👍

**Pull request merged**